Spinner/Spinner.swift

```diff
@@ -84,25 +84,23 @@ public class SpinnerView: NSObject, Spinner {
         let view = showSpinner(inView: button, style: style, color: color)
         
         if let spinnerView = view as? SpinnerView {
-            spinnerView.userInteractionEnabledAtReception = button.isUserInteractionEnabled
```

Comment:
@andrewlloyd100  Ah, I see. Ok then :) Can you verify this thing also works for custom spinner?